<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serialClass</class>
 <widget class="QWidget" name="serialClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>46</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,10,3">
   <item>
    <widget class="QPushButton" name="buttonRefresh">
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxSerials"/>
   </item>
   <item>
    <widget class="QPushButton" name="buttonConnect">
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
